### ISSUE TITLE
Log to file

### ISSUE THREAD

**Issue by conradludgate** [ellie/atuin#87]
Update our logger to write to a file as well as stdout to help with debugging. (see #79)

**Comment by ellie** [comment]
Yup this would be great! And remember that when the TUI runs, we swap stdout and stderr as a horrible hack to make things work properly

**Comment by tamasfe** [comment]
I suggest disabling stdout/stderr logging altogether when atuin wraps commands as well.

Setting the `RUST_LOG` variable will cause atuin logs to appear after every command. I was debugging an application which also used tracing for logs and seeing SQL logs after each command even when my app was not running was pretty scary :D .

To see this behavior (at least in zsh):

```sh
export RUST_LOG=info
echo "here be logs"
```
